lib/jekyll/static_file.rb

```diff
@@ -134,7 +134,7 @@ def url
       @url ||= if @collection.nil?
                  relative_path
                else
-                 ::Jekyll::URL.new(
+                 URL.new(
```

Comment:
This was just to make sure we get the Jekyll::URL. Generally speaking I like to use the fully-qualified class name to reduce ambiguity.